feconf.py

```diff
@@ -571,6 +581,9 @@ def get_empty_ratings():
 LIBRARY_TOP_RATED_URL = '/library/top_rated'
 NEW_COLLECTION_URL = '/collection_editor_handler/create_new'
 NEW_EXPLORATION_URL = '/contributehandler/create_new'
+NEW_SKILL_URL = '/contributehandler/create_new_skill'
```

Comment:
This might actually be /topichandler/{{topicID}}/create_new_skill since skills are generally created within the context of topics. Not sure, but we should be able to change it later anyway.

In fact, why add these now if they're not used? Add them when you're doing the controllers.

Reply:
Done, removed them for now.